src/chapters/orm-relationships/exercises.rst

```diff
@@ -0,0 +1,24 @@
+Exercises: Early Bird gets the ORM!
+===================================
+
```

Comment:
Since these exercises are focused on what the student needs to complete, rather than the LC staff, I'd vote to change most of the "we" references to "you".

Reply:
Done!